.github/workflows/bump-version-tag.yml

```diff
@@ -1,28 +1,50 @@
-name: Bump Version and Tag for Release
+name: Conditional Bump Version
 
 on:
-  workflow_dispatch:
-    inputs:
-      bumpversion:
-        description: "Bumpversion"
-        required: true
-        default: "patch"
-        options:
-            - "patch"
-            - "minor"
-            - "major"
-      tag:
-        description: "Tag"
-        required: true
-        default: true
-        options:
-            - "true"
-            - "false"
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - .*
+      - .github/*/*.md
+      - .github/*/*.yml
+      - AUTHORS.rst
+      - CHANGES.rst
+      - Dockerfile
```

Comment:
This `paths-ignore` section is to not bump if those files listed here change?  Why would we not bump and make a release if `Dockerfile` changes?  Same for `environment.yml`.  Those files are part of the release docker build.

Reply:
I can remove the `environment.yml` files, no problem there.

The reason for not including the `Dockerfile` is because the version string is changed in the `Dockerfile` on `bump-my-version`, which would trigger `bump-my-version` to augment the version, which would cause changes in the `Dockerfile`, which would trigger `bump-my-version` to augment the version, which would cause changes in the `Dockerfile`...

Reply:
I see, to avoid infinite loop. Okay, got that, good with leaving `Dockerfile` out.